src/server/ApiRoutes/Bib.js

```diff
@@ -34,6 +34,40 @@ const findUrl = (location, urls) => {
   return longestMatch.url;
 };
 
+const checkInItemsForHolding = (holding) => {
+  let location = '';
+  let holdingLocationCode = '';
+  if (holding.location.length) {
+    holdingLocationCode = holding.location[0].code;
+    location = holding.location[0].label;
+  }
+  const format = holding.format || '';
```

Comment:
We should test that the correct format is shown in the "Format" column. I believe this format can be different from the bib format, which is why we are adding the "Format" filter. There is skipped test for this in the `BibPage` unit test. Maybe it could go somewhere else.

Reply:
Added those tests in